### PR TITLE
[adguard-home] Initial Commit

### PR DESCRIPTION
**Pull request by dcplaya**
#### Special notes for your reviewer:
Similar functionality to Pi-Hole/Blocky

Has options for DHCP and DNS-over-TLS but those have not been tested since my LAN isn't set up to test that. During the initial setup, Adguard allows the HTTP port to be changed from the default 3000, this can't be changed. There might be a more elegant solution but I couldn't think of one. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/adguard-home/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+## Upgrading an existing Release to a new major version
+
+A major chart version change (like 2.2.2 -> 3.0.0) indicates that there is an
+incompatible breaking change needing manual actions.
+
+### Upgrading from 2.x.x to 3.x.x
+
+Due to the renaming of the service port, an upgrade-in-place will not work.  The following are possible approaches to solve this:
```
Comment (billimek) [inline charts/adguard-home/README.md:57]: Let's remove this section until/unless we have to deal with a breaking change and specific instructions for that.

File: charts/adguard-home/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+helm upgrade --force
+```
+
+#### Delete the existing `adguard-home` services prior to upgrading
+
+```sh
+kubectl delete svc/adguard-home
+```
+
+#### Remove the existing blocky chart first
```
Comment (billimek) [inline charts/adguard-home/README.md:71]: This should be `adguard`

File: charts/adguard-home/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+This is the 'easiest' approach, but will incur downtime which can be problematic if you rely on blocky for DNS
```
Comment (billimek) [inline charts/adguard-home/README.md:73]: This should be `adguard`

File: charts/adguard-home/README.md (hunk 1 above)
Comment (dcplaya) [reply charts/adguard-home/README.md]: Fixed them all in the newest commit.

**Pull request merged**